### PR TITLE
Added check for NeutralResourceslanguage attribute

### PR DESCRIPTION
**Pull request by ajaybhargavb**
Issue - #3 
@Eilon @pranavkm

### CODE REVIEW

File: src/NuGetPackageVerifier/Rules/AssemblyHasNeutralResourcesLanguageAttributeRule.cs [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+                    extension.Equals(".exe", StringComparison.OrdinalIgnoreCase))
+                {
+                    var assemblyPath = Path.ChangeExtension(
+                        Path.Combine(Path.GetTempPath(), Path.GetTempFileName()), extension);
+
+                    try
+                    {
+                        using (var packageFileStream = currentFile.GetStream())
+                        {
+                            var _assemblyBytes = new byte[packageFileStream.Length];
```
Comment (pranavkm) [inline src/NuGetPackageVerifier/Rules/AssemblyHasNeutralResourcesLanguageAttributeRule.cs]:
```C#
using (var packageFileStream = currentFile.GetStream())
using (var fileStream = new FileStream(assemblyPath, FileMode.Create))
{
   packageFileStream.CopyTo(fileStream);
}
```

Shouldn't need to copy it to a stream

File: src/NuGetPackageVerifier/Rules/AssemblyHasNeutralResourcesLanguageAttributeRule.cs [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+                    }
+                }
+            }
+
+            yield break;
+        }
+
+        private static bool HasNeutralResourcesLanguageAttribute(Mono.Collections.Generic.Collection<CustomAttribute> asmAttrs)
+        {
+            var validAttributes = asmAttrs.Where(asmAttr => IsValidNeutralResourcesLanguageAttribute(asmAttr));
```
Comment (pranavkm) [inline src/NuGetPackageVerifier/Rules/AssemblyHasNeutralResourcesLanguageAttributeRule.cs]: `return asmAttributes.Any(IsValidNeutralResourcesLanguageAttribute);`

**Comment by pranavkm** [comment]
:shipit:

**Comment by Eilon** [comment]
:shipit: after @pranavkm 's comments.

**Pull request merged**